content/_data/changelogs/lts.yml

```diff
@@ -2062,10 +2062,173 @@
       pull: 3614
       message: >
         Improve robustness when search items don't specify a display name.
+    - type: rfe
+      references:
+      - pull: 3690
+      - url: https://github.com/stapler/stapler/pull/149
+        title: stapler/stapler#149
+      message: >
+        Update Stapler from 1.254.2 to 1.255 to integrate a fix related to <code>StaplerProxy#getTarget()</code> return value handling.
     - type: rfe
       issue: 54029
       message: >
         Add telemetry trial related to Stapler request dispatching.
+- version: 2.138.4
+  date: 2018-12-05
+  changes:
+    - type: security
+      message: Important security fixes.
+      references:
+      - url: https://jenkins.io/security/advisory/2018-12-05/
+        title: security advisory
+- version: 2.150.1
+  date: 2018-12-05
+  lts_predecessor: "2.138.4"
+  lts_baseline: "2.150"
+  lts_changes:
+    - type: major rfe
+      message: >
+        Numerous fixes and improvements to better support running Jenkins on Java 11.
+      references:
+      - issue: 46523
+      - issue: 46725
+      - issue: 51805
+      - issue: 52019
+      - issue: 52024
+      - issue: 53693
+      - issue: 53710
+      - issue: 53929
+      - url: https://github.com/jenkinsci/pom/blob/master/CHANGELOG.md#149
+        title: Parent POM 1.49 changelog
+      - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#51
+        title: Winstone-Jetty 5.1 changelog
+      - url: http://groovy-lang.org/changelogs/changelog-2.4.12.html
+        title: Groovy 2.4.12 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.12.v20180830
+        title: Jetty 9.4.12 changelog
+      # pull: 3637
+      # pull: 3641
+      # pull: 3643
+      # pull: 3650
+      # pull: 3659
+      # pull: 3660
+      # pull: 3682
+      # Fix problems with update center metadata signature check on Java 11.
+      # Update Groovy from 2.4.11 to 2.4.12 to pick up fixes towards Java 11 support.
+      # Update jnr-posix to 3.0.45 to prevent Illegal Reflective access warnings when running with Java 11.
+      # Update Unix process management logic to support Process Tree termination when running with Java 11.
+      # Update Winstone-Jetty from 5.0 to 5.1 to update Jetty to 9.4.12, picking up TLS 1.3 support and fixes towards Java 11.
+      # Developer: Update PowerMock and Mockito to versions compatible with Java 11.
+      # Internal: Update META-INF/services generator from 1.4 to 1.8 to fix compilation on JDK 10+.
+      # Internal: Update Parent POM to 1.49 to make the build flow compatible with Java 11.
```

Comment:
Seemed preferable to do one entry with all of them rather to do these all individually.